### ISSUE TITLE
ImportError: No module named 'bids'

### ISSUE THREAD

**Issue by pn2200** [jmtyszka/bidskit#64]
**Describe the bug**
When I run `docker run -it -v /dataset/:/dataset jmtyszka/bidskit bidskit -d /dataset`, there is a ImportError:

```
Traceback (most recent call last):
  File "/usr/local/bin/bidskit", line 11, in <module>
    load_entry_point('bidskit==2019.8.16', 'console_scripts', 'bidskit')()
  File "/usr/lib/python3/dist-packages/pkg_resources/__init__.py", line 561, in load_entry_point
    return get_distribution(dist).load_entry_point(group, name)
  File "/usr/lib/python3/dist-packages/pkg_resources/__init__.py", line 2631, in load_entry_point
    return ep.load()
  File "/usr/lib/python3/dist-packages/pkg_resources/__init__.py", line 2291, in load
    return self.resolve()
  File "/usr/lib/python3/dist-packages/pkg_resources/__init__.py", line 2297, in resolve
    module = __import__(self.module_name, fromlist=['__name__'], level=0)
  File "/usr/local/lib/python3.5/dist-packages/bidskit-2019.8.16-py3.5.egg/bidskit/__main__.py", line 42, in <module>
  File "/usr/local/lib/python3.5/dist-packages/bidskit-2019.8.16-py3.5.egg/bidskit/translate.py", line 36, in <module>
ImportError: No module named 'bids'
```

**Comment by dmd** [comment]
I am still getting this with a fresh pull.

```
$ docker run -it -v /data/ddrucker/j/:/dataset jmtyszka/bidskit bidskit -d /dataset
Unable to find image 'jmtyszka/bidskit:latest' locally
latest: Pulling from jmtyszka/bidskit
9cc2ad81d40d: Pull complete
e6cb98e32a52: Pull complete
ae1b8d879bad: Pull complete
42cfa3699b05: Pull complete
053cac798c4e: Pull complete
e11ff976ff71: Pull complete
81d72bab54a7: Pull complete
fcefcb4d99e7: Pull complete
592d9f42009b: Pull complete
b6676746da9a: Pull complete
6c090567735a: Pull complete
ad24136173b8: Pull complete
4a7f1ec866ef: Pull complete
41d9fc8bcabe: Pull complete
Digest: sha256:4485d8b5c9087799b33ab6d03be4c9e575675694e73c0325c848c6b54746f712
Status: Downloaded newer image for jmtyszka/bidskit:latest
Traceback (most recent call last):
  File "/usr/local/bin/bidskit", line 11, in <module>
    load_entry_point('bidskit==2019.8.16', 'console_scripts', 'bidskit')()
  File "/usr/lib/python3/dist-packages/pkg_resources/__init__.py", line 561, in load_entry_point
    return get_distribution(dist).load_entry_point(group, name)
  File "/usr/lib/python3/dist-packages/pkg_resources/__init__.py", line 2631, in load_entry_point
    return ep.load()
  File "/usr/lib/python3/dist-packages/pkg_resources/__init__.py", line 2291, in load
    return self.resolve()
  File "/usr/lib/python3/dist-packages/pkg_resources/__init__.py", line 2297, in resolve
    module = __import__(self.module_name, fromlist=['__name__'], level=0)
  File "/usr/local/lib/python3.5/dist-packages/bidskit-2019.8.16-py3.5.egg/bidskit/__main__.py", line 42, in <module>
  File "/usr/local/lib/python3.5/dist-packages/bidskit-2019.8.16-py3.5.egg/bidskit/translate.py", line 36, in <module>
ImportError: No module named 'bids'
```

**Comment by jmtyszka** [comment]
Fixed in v2019.12.20
